src/helm/proxy/clients/together_client.py

```diff
@@ -12,6 +13,12 @@
 from .client import Client, wrap_request_time, truncate_sequence
 
 
+MODEL_ALIASES = {
+    "bloomz": "bloomz-176b-alpa",
```

Comment:
This is fine, but it's worth noting that the exact Together model will have an impact on efficiency metrics.

Reply:
Actually I do wonder if we want to use the actual Together names just to be completely transparent...

Reply:
I don't like the idea of using Together names because they haven't been particularly stable. Most of the current Together model names are already stale. Also I don't think it makes sense to have implementation details in the name that we show users.

Reply:
The Together names should be (more) stable now.  I think it's easier to merge than to separate later.  If we ever need to distinguish between different implementations, we need to separate it out (certainly will be different for efficiency, but I think the models might not provide exactly the same predictions).

I agree it'd be nice to have simpler names for users - we can perhaps use aliases on our side which resolve immediately to an implementation.

Reply:
How about:

- User sees "together/bloomz"
- Cache key and raw request both use "bloomz-176b-alpa"

Then if the Together implementation changes name, it will automatically invalidate the cache.

I'm thinking about the H3 model which is named "h3-2.7b-h3" (see #1404), which just seems strange to expose to users.

Reply:
That works - the caching of the underlying model makes me feel better about this.  And if there is a change, we can always migrate..  Ideally when the user makes a request, it will map `together/bloomz` to `together/bloomz-176b-alpa` (or whatever is the version de jour), and the user could could also request `together/bloomz-176b-alpa` to get particular implementations if they want.